javascript/javascript2/week3/function-variable.js

```diff
@@ -0,0 +1,33 @@
+//Functions as a Variable
+
+function function_one() {
+  console.log("this is first element in array");
+}
+function function_two() {
+  console.log("this is second element in array");
+}
+function function_three() {
+  console.log("this is third element in array");
+}
+const funArray = [function_one(), function_two(), function_three()];
+for (let i = 0; i < funArray.length; i++) {
+  funArray[i];
+}
```

Comment:
I think it is more correct to store function's names in the array, and then call it inside for loop.
```
const funArray = [function_one, function_two, function_three];
for (let i = 0; i < funArray.length; i++) {
  funArray[i]();
}
```
You could also used forEach method for this task, since you are already learned it :)

Reply:
Thank you @OlkaA for feedback . I will fix it.